geniza/corpus/templates/corpus/snippets/document_transcription.html

```diff
@@ -1,4 +1,61 @@
 {% load i18n corpus_extras %}
+<section id="transcription-panel">
+    {# NOTE: inputs must be kept as SIBLINGS of panel-container for CSS selection/controls #}
+    {# images displayed by default; disable if no images are available #}
+    <input type="checkbox" id="images-on"  {% if document.has_image %}checked="true" {% else %} disabled="true"{% endif %}/><label for="images-on">show images</label>
+    {# transcription displayed by default; disable if no content is available #}
+    <input type="checkbox" id="transcription-on" {% if document.digital_editions %}checked="true" {% else %} disabled="true"{% endif %}><label for="transcription-on">show transcription</label>
+    {# translation disabled for now since we don't handle them in the system yet #}
+    <input type="checkbox" id="translation-on" disabled="true"/><label for="translation-on">show translation</label>
+    <div class="panel-container">
+        {% comment %}
+NOTE: if we have more transcription pages than images,
+they won't show in current implementation.
+May want to gather them in the model... (but transcription content format will change)
```

Comment:
yeah... I'm worried that there will be cases in the current implementation where we're not showing all the available content. But I also don't want to invest a lot of effort in this version of the transcription structure since we're actively working on changing it. 

Maybe all we need is a way to get the maximum number of panel rows (images or transcription chunks) to ensure that we display all available content, and iterate on that range. If that can be done in the templates, that would be preferable vs adding more logic to the model.

Reply:
I can't think of a way to do it in the template alone, but we could add a template filter that would pick the longer of the two lists and use `with` to set it as the iterator. Would a template filter or model logic be better?